### DISCUSSION TITLE
Work account suspended

### DISCUSSION THREAD

**Discussion by zsolt-szabo** [community/community#116569] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My work account was suspended and all my github contributions outside of the company has been wiped from github (Releases, issue, PR contributions).  This is not only catastrophic for me, but my company is now missing documentation that is necessary.

The cause of the suspension is unknown.  I received no emails and believe I am not doing anything harmful to my knowledge.  I'm relatively sure I am within the throttle limit as well, but I would expect a throttle cut off and not an account suspension anyway.

**Comment by gpereiradev** [comment]
I have the same issue

**Reply by arvindrag** [reply]
Same issue with work associated account. Clearly a widespread issue - but the lack of communication from github is disappointing. Some folks have had success with password reset - but I keep getting account suspended.
My org uses github as auth for Circle-CI - some suspicion that this might be related - anyone else?

**Comment by paulaustin-automutatio** [comment]
I also had the same issue, I tried resetting the password again after seeing this and it worked.

Very disappointed that there was no notifications as to why the account was suspended. No alerts if there was a problem. And no response within 2 hours to the support ticket.

**Comment by gpereiradev** [comment]
try logging in again, it worked

**Comment by paulaustin-automutatio** [comment]
Further on this when you get back in have a look at:

https://github.com/settings/security-log

at 09:58 (PST) there was a user.suspend entry from github-staff, followed by expiry of a whole bunch of access tokens.

Then at 10:40 PST there was a user.unsuspend from github-staff.

No email notifications.

Anyone see similar?

**Reply by arvindrag** [reply]
yup - same - unsuspended

**Reply by arvindrag** [reply]
no emails

**Reply by pooksh** [reply]
i had a whole bunch of staff revoked tokens first, and then a suspension, then an immediate unsuspension

**Comment by zsolt-szabo** [comment]
Github has reinstated my work account.  From support request to problem solved was about 4 hours.  They neither emailed me, nor within the logs provide any explanation for the suspension.   Someone suggested an intern having a bad day :-).

**Comment by github-staff** [comment]
Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated.
We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

**Reply by mavrw** [reply]
I have been affected by being falsely flagged for spam and it is preventing my ability to work.

**Reply by XuanCiTech** [reply]
@github-staff 

Hello staff, I am appealing and requesting restoration for my account.

https://github.com/League2EB

This account received an Account suspended without any prior notice.

Things that can be confirmed:

1. My account did not violate any terms of use, as usual developing Swift, Python, JS.

2. Did not create or participate in the development of any DCMA repositories.

3. The application for github education was made using my school identity document.

I firmly believe this is a mistaken judgment for suspension, as there are absolutely no illegal activities involved.

At the same time, an appeal was made through the form Ticket ID: 3085227.

The unconditional Account suspended has caused great inconvenience and obstruction in work and assignments, making it impossible to carry out any tasks.

It is very urgent to restore my account to continue my studies.

If it is due to issues with the github education application documents, please let me know what additional requirements and documents are needed.

Thank you very much @github-staff.

**Reply by ErNewDev0** [reply]
> Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated. We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

hi @github-staff my account [ErRickow](https://github.com/ErRickow) is being suspended with no reason, no email notification and the account is still active at 8am GMT Jakarta.

Please Restore My Account, there is so much work on their and is now disappearing with no reason. I currently active on github every day, thats cut off my deep dream . Please Response This Isssue

**Comment by alanzimm85** [comment]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).

**Comment by vallari-sharma** [comment]
@github-staff
I have my job interviews from next week and my account (valls-25) associated with vallarisharma25@gmail.com has been suspended for a reason unbeknownst to me . I assure you I have not violated any policy.
 I issued a ticket (3422317) , however , currently if I make any new Git accounts with a new email and try to commit my codes , it says that my account is suspended. There is no way through at all for me , no matter if I make a new Git account .
I plead to you to restore and unsuspend my account , because it has all my important repositories .
